src/main/java/uk/gov/ons/ctp/common/state/BasicStateTransitionManager.java

```diff
@@ -36,8 +38,10 @@ public S transition(final S sourceState, final E event) throws CTPException {
     Map<E, S> outputMap = transitions.get(sourceState);
     if (outputMap != null) {
       destinationState = outputMap.get(event);
+      log.info("Moving from " + sourceState.getClass().getName() + " to " + destinationState.getClass().getName() + " due to " + event.getClass().getName());
     }
     if (destinationState == null) {
```

Comment:
I think the destinationState being null is quite a normal case and this will probably lead to lots of errors in the logs. Is this something we want?

Reply:
Caught this in the Travis build and have fixed :)